lib/appsignal/cli/install.rb

```diff
@@ -281,7 +281,7 @@ def installed_frameworks
         def framework_available?(framework_file)
           require framework_file
           true
-        rescue LoadError
+        rescue LoadError, NameError
```

Comment:
Would it work to check for the known error messages here so we don't lose too much accuracy?

Reply:
On one hand I would like to rescue all errors and consider any kind of error a problem for the installer to ignore, but in this case it's just Padrino. Once they release a new version we could even remove this rescue.